lib/recite.rb

```diff
@@ -2,20 +2,28 @@
 require 'pry-byebug'
```

Comment:
issue: this should be removed from checked-in code. it can be added back when debugging.